src/constants.js

```diff
@@ -4,65 +4,83 @@ import { BigInt } from "jsbi";
 export const BASE_FACTOR = 1000000n;
 export const BLOCKS_PER_ERA = 8000000n;
 export const NUMBER_OF_ERAS = 8n;
-export const NETWORK_ID = 1_793_045_504; // hello-world-0
+export const NETWORK_ID = 1;
 export const PROD = process.env.NODE_ENV === "production";
-export const ETH_BRIDGE_ADDRESS = "0x861B3289E2432138324aB593090E0805f9337DAe";
 export const LIQUIDITY_FEE = 3000n;
 export const ZERO = 0n;
-export const BRIDGE_ADDRESS = base64url.toBuffer(
-  "OaKmwCWrUhdCCsIMN_ViVcu1uBF0VM3FW3Mi1z_VTNs"
-);
 export const BOOTNODES = PROD
   ? ["davenport.ellipticoin.org"]
   : ["localhost:8080"];
 export const ELC = {
   ticker: "ELC",
   name: "Ellipticoin",
-  issuer: "Ellipticoin",
-  id: Buffer.from("ELC", "utf8").toString("base64"),
+  address: Buffer.from(
+    "0000000000000000000000000000000000000001",
+    "hex"
+  ).toString("base64"),
```

Comment:
It seems the address is becoming the primary key for coins, no?

Reply:
yep! Another added benefit of transitioning to ETH addresses 👍